stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/theme/StreamDimens.kt

```diff
@@ -157,6 +161,8 @@ public data class StreamDimens(
             quotedMessageAttachmentTopPadding = 6.dp,
             quotedMessageAttachmentStartPadding = 8.dp,
             quotedMessageAttachmentEndPadding = 0.dp,
+            groupAvatarInitialsXOffset = 1.5.dp,
```

Comment:
True, but maybe the users change the font size as well and need to position it in a little bit different. I think we should expose it so if they customise other parameters they can customise this one as well so it would fit their use as well.